src/app/common/services/configuration-service.coffee

```diff
@@ -0,0 +1,14 @@
+angular.module("doubtfire.common.services.configuration", [])
```

Comment:
If we remove the configuration service, we could place this under a `config/external-name/external-name.coffee` file. Rename the module according to the folder structure (`doubtfire.config.external-name` in this case). You can see how we do the `contributors` constant under `contributors.coffee`.